docs/courses/csintro1/loops/intro.md

```diff
@@ -110,13 +111,13 @@ pause(100)
 
 ## Student Task #1: Move to the upper left corner, and move faster
 
-1. Load up the blocks from the previous example
+1. Reuse the code from the previous example
 2. Make the sprite move up and to the left instead - to do so, change all movements to be in the opposite direction
 3. Change the ``||loops:pause||`` between each step to be only 50 ms, instead of 100 - we decided we want the ghost to be a little bit faster than it was
 
 ## Concept: Moving a sprite using the repeat block loop
 
-When completing the last task, you likely noticed that you were doing the same action repeatedly - moving in one direction, pausing, moving in another, pausing, and then repeating that. Instead of doing that by inserting the same chunk of code multiple times, we can, by using loops, repeat that chunk of code more easily.
+When completing the last task, you likely noticed that you were doing the same action repeatedly - moving in one direction, pausing, moving in another, pausing, and then repeating that. Instead of doing that by inserting the same chunk of code multiple times, we can, by using loops, ``||loopps:repeat||`` that chunk of code more easily.
```

Comment:
```suggestion
When completing the last task, you likely noticed that you were doing the same action repeatedly - moving in one direction, pausing, moving in another, pausing, and then repeating that. Instead of doing that by inserting the same chunk of code multiple times, we can, by using loops, ``||loops:repeat||`` that chunk of code more easily.
```